src/components/eventing/FilterForm.jsx

```diff
@@ -0,0 +1,139 @@
+import React, {useEffect, useState} from "react"
+import { Modal, Form, Select, Checkbox, DatePicker, Icon} from 'antd';
+import FormItemLabel from "../form-item-label/FormItemLabel"
+import client from "../../client";
+//redux
+import {useDispatch, useSelector} from "react-redux";
+import {set, increment, decrement, reset} from "automate-redux";
+
+import moment from 'moment';
+
+const {Option} = Select;
+const { RangePicker } = DatePicker;
+
+const FilterForm = (props) => {
+  const dispatch = useDispatch();
+  const eventFilters = useSelector(state => state.uiState.eventFilters);
+  const triggerNames = useSelector(state => state.uiState.triggerNames);
+  
+  useEffect(() => {
+    dispatch(increment("pendingRequests")); 
+    client.eventing.fetchTriggerNames(props.projectID)
```

Comment:
We don't need to fetch trigger names as they are already there inside redux in `modules.eventing.rules` inside project config